rocAL/rocAL/source/loaders/video/video_read_and_decode.cpp

```diff
@@ -141,6 +141,12 @@ void VideoReadAndDecode::decode_sequence(size_t sequence_index)
     }
 }
 
+size_t
+VideoReadAndDecode::last_batch_padded_size()
+{
+    return 0;
```

Comment:
why 0 is returned?

Reply:
For video loader , the support for last batch policy is not added.